packages/api/lambdas/sf-event-sqs-to-db-records/index.js

```diff
@@ -87,14 +94,20 @@ const writeRecords = async (cumulusMessage, knex) => {
       knex,
       executionCumulusId,
     });
-    return await writeGranules({
+    await writeGranules({
       cumulusMessage,
       collectionCumulusId,
       providerCumulusId,
       executionCumulusId,
       pdrCumulusId,
       knex,
     });
+    return await writeRules({
```

Comment:
I know that this is a WIP - but I just glanced at this and wanted to call out that rules are not one of the data types that is written to the database by the lambda. So the logic for saving them wouldn't go here, nor is there ever (that I'm aware of) rules data in `payload.rules`. Rules are created only via the API in `packages/api/endpoints/rules.js`, so we need to update that code to save to both DynamoDb and Postgres. We have already done for this collections if you want to look at that API endpoint code.

Reply:
Thanks, I opened it as a draft _because_ I wasn't really sure I was going down the right path. I will take a look at the collections api endpoint code. I for some reason assumed that rules would be in the payload of a cumulus message but I can't remember how I came to that conclusion.